src/Dollar.cpp

```diff
@@ -1,12 +1,9 @@
-// Dollar.cpp
-#include <iostream>
-#include "Dollar.h"
-
-Dollar::Dollar(){
-	std::cout << "Constructor invoked." << std::endl;
+#include "Dollar.h"                                                                                                               
+Dollar::Dollar(int amount){
+	this->amount= amount;
```

Comment:
Please always put a space before and after `=` sign, just for the matter of readability.